client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/types.ts

```diff
@@ -12,7 +13,8 @@ export interface NavigatorLocation {
 }
 
 export type Category = {
-	name: string;
+	slug?: string;
```

Comment:
The PTK API uses `category.slug` but the categories API `/block-patterns/categories` uses `category.name`.